src/chainstate/stacks/boot/pox-2.clar

```diff
@@ -898,6 +898,7 @@
             data: {
                 increase-by: increase-by, 
                 total-locked: new-total-locked,
```

Comment:
The `event-info` here is "stale" by the time this function returns, for example:
```
  locked: 2062698000000000n,
  balance: 7937301999980000n,
  burnchain_unlock_height: 115n,
  name: 'stack-increase',
  data: { increase_by: 123n, total_locked: 2062698000000123n }
```
The `locked` value from `event-info` shows the value from before this function was called. The accurate value is in `total_locked` but these seems like inconsistent and surprising behavior.